hyp3_autorift/vend/testautoRIFT.py

```diff
@@ -539,6 +550,17 @@ def generateAutoriftProduct(indir_m, indir_s, grid_location, init_offset, search
                     preprocessing_methods[ii] = 'wallis_fill'
             elif len(re.findall("LT0[45]_", name)) > 0:
                 preprocessing_methods[ii] = 'fft'
+        
+        zero_mask = None
+        indir_m_zero = f'{indir_m.split(".")[0]}_zeroMask.{indir_m.split(".")[1]}'
+        indir_s_zero = f'{indir_s.split(".")[0]}_zeroMask.{indir_s.split(".")[1]}'
+        if os.path.exists(indir_m_zero) or os.path.exists(indir_s_zero):
+            m_zero, s_zero = loadProductOptical(indir_m_zero, indir_s_zero)
+            m_zero = m_zero.astype(np.uint8)
+            s_zero = s_zero.astype(np.uint8)
+
+            # FIXME: Or? Wallis uses "or" here, while wallis_fill uses "and" here.
```

Comment:
Also, I think `and` vs `or` is important here. Could be why we're getting slightly less data than `develop currently.

Reply:
I've decided to switch to `and` for now.